bot/src/main/java/com/community/tools/model/Mentors.java

```diff
@@ -16,8 +17,11 @@ public class Mentors {
 
   @Id
   private String gitNick;
+  @Column(unique = true)
+  private String discordName;
```

Comment:
is this disc name or id? As far as i know disc allows name duplications

Reply:
In a somewhat recent [change](https://support.discord.com/hc/en-us/articles/12620128861463-New-Usernames-Display-Names#docs-internal-guid-28d8eed3-7fff-7f1c-a2e6-21318f925157) usernames were made unique. The problem here may arise from the fact that a mentor can change their username and we may lose the ability to notify them on discord if we keep only their username, but it may be more appropriate to consider the logic around this while implementing addition/registration of mentors to/in the system at large.